extensions/vscode/src/commands/tutorialkit.delete.ts

```diff
@@ -0,0 +1,18 @@
+import { cmd } from '.';
+import * as vscode from 'vscode';
+import { Node } from '../models/Node';
+import { getLessonsTreeView } from '../global-state';
+
+export async function deleteUnit(selectedNode: Node | undefined, selectedNodes: Node[] | undefined) {
+  let nodes: readonly Node[] = (selectedNodes ? selectedNodes : [selectedNode]).filter((node) => node !== undefined);
+
+  if (nodes.length === 0) {
+    nodes = getLessonsTreeView().selection;
+  }
+
+  for (const node of nodes) {
+    await vscode.workspace.fs.delete(node.path, { recursive: true });
+  }
+
+  return cmd.refresh();
+}
```

Comment:
Should this also delete the unit from metadata? Right now when you delete a lesson it is still present in custom ordered chapter metadata. 

For example, remove `One` from the tree view in this case. Should the `lessons` be modified here as well?

```md
---
type: chapter
title: Example
lessons:
  - One
  - Two
---
```

Reply:
Oh really good catch! 😍 Will make sure the parent is updated 👍